### DISCUSSION TITLE
Welcome to GitHub Mobile product feedback!

### DISCUSSION THREAD

**Discussion by candyho** [community/community#3731] [Mobile]
### Welcome! 👋

Have ideas or feedback for how we can further improve GitHub Mobile📱? We are all ears! 

This is the place for you to make feature suggestions or react to existing ones. Open a discussion in the mobile category and we'll review your suggestions. We'd love to hearing your feedback and are eager to continue in conversations with you! ❤️

For guidance on how to submit suggestions please refer to the [README](https://github.com/github/feedback#github-public-feedback-discussions) and our [code of conduct](https://github.com/github/feedback/blob/main/CODE_OF_CONDUCT.md).

**Comment by eduarau** [comment]
Good

**Comment by kolumnin** [comment]
Will it add to network interruptions?

**Reply by 62f** [reply]
Define ```"it"```
![image](https://github.com/community/community/assets/68332197/beb8d191-71e1-4ca7-9b94-f3c78808aa0f)

**Comment by sudo-nautilus** [comment]
Noice ++ love the app. But I want to commit to repositories from the app

**Comment by kalsangdorjee** [comment]
Please add a confirmation dialog which should let the user conform to close an issue when he/she click the `close` button. I faced this issue when I mistakenly tapped the `close` button in one of mine opened issue

**Comment by Alfaksadi** [comment]
I like the app

**Comment by grab-a-byte** [comment]
I'd like the ability to be able to view files over 10MB, whether this is a button to see the file or otherwise, can't just casually download repos on mobile.

**Comment by tentacleuno** [comment]
Where is the source code for the GitHub mobile application, please?

**Reply by 0xdevalias** [reply]
@tentacleuno While it's not answered, there is a specific discussion thread about this at https://github.com/github/feedback/discussions/2841 that you might like to check out.

**Comment by snoopiedoom** [comment]
when browsing code, i believe it would provide a much better experience if scrolling didnt get locked to a scrollbar, but to be able to move the view freely in any direction.

**Comment by tflynn3** [comment]
Render or preview python jupyter notebooks on mobile similar to on browser

**Comment by CBID2** [comment]
Hi @candyho and everyone else! :) I love the mobile app but I wish it had some of the same options as the desktop app(e.g., editing code lines right on the app and seeing your history on GitHub). It would make these easier users who are not on their computers often. Another feature I would like to be added is notifications for when someone follows or peeps at your profile (kind of like Linkedin).

**Reply by rosedalerk** [reply]
I agree that editing code lines right on the app would be useful.

**Comment by snoopiedoom** [comment]
When browsing code, for the love of god, let me scroll freely in any direction. Not being locked to either vertical or horizontal scroll is a must on a mobile device without line wrap.

**Comment by wavemoran** [comment]
When trying to unsubscribe from the TestFlight email notifications, I receive an email bounce for messages sent to `mobilefeedback@github.com`. 

```
The response was:
Message bounced due to organizational settings.
```

**Comment by CBID2** [comment]
I'd it if the mobile app notified you if someone follows you on GitHub. Something similar to X(Twitter), it'll be a great way for users to connect with each other.

**Reply by 62f** [reply]
You'd absolutely it.

**Comment by leki94** [comment]
opa

**Comment by prudhomm** [comment]
on android the mobile app does not support emoticon in labels, it makes the interface quite ugly if there are many emoticons used in labels. Would it be possible to support them ?

**Reply by 62f** [reply]
> on android the mobile app does not support emoticon in labels, it makes the interface quite ugly if there are many emoticons used in labels. Would it be possible to support them ?
> 
> 
What flavor\s of Android are\is giving you trouble.

**Comment by Tembero11** [comment]
Show repository commit count like on the web app.

**Comment by cocosolos** [comment]
Would like to see who submitted an Issue/PR from the list view, even just a profile pic.

**Comment by damiansan239** [comment]
Please, can you seperate github actions notifications to have one for failed and successful build?

**Comment by VOM7HC** [comment]
Hi @candyho, can your team add keyboard shortcut for code viewer? that will be good for tablet

**Comment by DavidCRushBlyat** [comment]
![Screenshot_20250627-130554788.png](https://github.com/user-attachments/assets/c81c4b04-8204-4c9f-9313-7b7282be8b38)

No comments seen, nor the README content.

**Comment by eduarau** [comment]
Bom dia

**Comment by EriKWDev** [comment]
Really enjoy the iPhone app widgets for PRs and the contribution plot :) Would be nice if there was a similar widget for my open issues / issues I track
